lib/web_ui/lib/src/engine/platform_views/content_manager.dart

```diff
@@ -249,5 +249,9 @@ DomElement _defaultFactory(
 }) {
   params!;
   params as Map<Object?, Object?>;
-  return domDocument.createElement(params.readString('tagName'));
+  final DomElement element = domDocument.createElement(params.readString('tagName'));
+  element.style
+    ..width = '100%'
+    ..height = '100%';
```

Comment:
I think you could still one-line-ize this:

```suggestion
  return domDocument.createElement(params.readString('tagName'))
    ..style.width = '100%'
    ..style.height = '100%';
```

(But I haven't tested my suggestion, I just remember writing similar code :P)

Reply:
Works like a charm!